demo/Vagrantfile

```diff
@@ -22,15 +24,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
-  config.vm.provision "shell", privileged: false, inline: <<SCRIPT
-    ## Uncomment these lines to update flynn before starting
-    # sudo apt-get update -qq
-    # sudo apt-get install -y flynn-host
-    # sudo flynn-host download /etc/flynn/version.json
-
-    sudo start flynn-host
+  config.vm.provider "virtualbox" do |v|
+    v.gui = true
```

Comment:
Ah, I see where you are coming from. I agree that for new users having no indicator that it is running can lead to forgetting about it. The current downside is that the window shows a login terminal that is basically useless, so we need to do something about that. A text or graphical message instead of the login prompt that explains how to access the dashboard would be one solution to this.